docs/docs/Templates/financial-report-parser.md

```
@@ -21,14 +21,20 @@
-## Run the memory chatbot flow
+## Run the financial report parser flow
 
-1. Add your OpenAI API key to the OpenAI model.
-2. Click the **Playground** button, and then click **Send**.
+1. Add your **OpenAI API key** to the **Language model** model component.
+	Optionally, create a [global variable](/configuration-global-variables) for the **OpenAI API key**.
+
+	1. In the **OpenAI API Key** field, click <Icon name="Globe" aria-hidden="True" /> **Globe**, and then click **Add New Variable**.
+	2. In the **Variable Name** field, enter `openai_api_key`.
+	3. In the **Value** field, paste your OpenAI API Key (`sk-...`).
+	4. Click **Save Variable**.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)